WhatsOnTap/wwwroot/js/beerList.js

```diff
@@ -0,0 +1,8 @@
+$(function () {
+    $('[data-toggle="popover"]').popover()
+    .popover({html:true})
+})
+
+$(function () {
+    $('[data-toggle="tooltip"]').tooltip()
```

Comment:
Can we just chain these event listeners together and only have them in one document ready call? 

`$('[data-toggle="popover"]').tooltip().popover().popover({html:true})` ?